CONTRIBUTING.md

```diff
@@ -1,10 +1,13 @@
+## Testing your PR in TeamCity
+
+For your PR to be honoured by TeamCity, its name needs to start with `3.x` if you plan to merge it to `3.x` and with `1.x` if you plan to merge it to `1.x`. (eg: `3.x/fix-something`).
```

Comment:
You read my mind, thanks for this change! Actually I would put the information about `1.x` in the README.md file of the `1.x` branch.

Pls change "For your PR" to "For your feature branch".